tests/gabbits/list-snippets-pagination.yaml

```diff
@@ -0,0 +1,256 @@
+common:
+  - &datetime_regex /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d+(\+\d{2}:\d{2})|Z$/
+  - &request_id_regex /^[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}$/
+  - &slug_regex /^[a-zA-Z0-9]+$/
+
+fixtures:
+  - XSnippetApiWithSnippets
+
+tests:
+  - name: get recent snippets
+    GET: /v1/snippets
+    query_parameters:
+      limit: 4
+    response_headers:
+      content-type: application/json
+      x-request-id: *request_id_regex
+    response_link_header:
```

Comment:
@malor I'd appreciate if you can double check `response_link_header` in these test cases (all indexes and markers).

Reply:
LGTM!